tools/execute-selenium-tests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_HOME="$CURRENT_PATH/.."
+FOXBOX_BINARY="$PROJECT_HOME/target/debug/foxbox"
+
+DEBUG_HOST=${BOX_LOCAL_NAME:-'localhost'}
+DEBUG_PORT=${BOX_PORT:-3331}
+DEBUG_PROFILE="$HOME/.local/share/foxbox-debug"
+
+# clean up residual profile
+rm -rf ${DEBUG_PROFILE}
+
+# build and launch foxbox daemon
+pushd ${PROJECT_HOME}
+cargo build
+$FOXBOX_BINARY -l ${DEBUG_HOST} -p ${DEBUG_PORT} --disable-tls --profile ${DEBUG_PROFILE} &
+FOXBOX_PID=$!
+popd
+
+# run tests
+BOX_PORT=${DEBUG_PORT} ${PROJECT_HOME}/node_modules/.bin/mocha ${PROJECT_HOME}/test/selenium/*_test.js
```

Comment:
Nit: Double-quotes on `${PROJECT_HOME}/node_modules/.bin/mocha` and `${PROJECT_HOME}/test/selenium/`.